text/0233-opentelemetry-sandbox.md

```diff
@@ -0,0 +1,89 @@
+
+As a possible solution to this, this OTEP proposes a new GitHub organization, [opentelemetry-sandbox](https://github.com/opentelemetry-sandbox). This organization will host projects until there's confidence that they have a healthy community behind them. They would also serve as a neutral place for the community to conduct experiments.
+
+The advantage of a sandbox organization is that OpenTelemetry's governance rules can be applied, making sure it’s an inclusive place for people to collaborate while keeping the reputation of the OpenTelemetry project as a whole untouched, given that it would be clear that OpenTelemetry doesn’t officially support projects within the sandbox.
+
+There is a desire, but not an expectation, that projects will be moved from the sandbox as an official SIG or incorporated into an existing SIG. There’s also no expectation that the OpenTelemetry project will provide resources to the sandbox project, like extra GitHub CI minutes or Zoom meeting rooms, although we might evaluate individual requests.
+
+Projects in the sandbox should leave it within one year, either by being merged into existing SIGs, being accepted as independent SIGs, or by being archived.
+
+This OTEP is inspired by [CNCF’s sandbox projects](https://www.cncf.io/sandbox-projects/), but the process is significantly different.
+
+## Internal details
+
+### Acceptance criteria
+
+A low barrier to entry would be desired for the sandbox. While the process can be refined based on our experience, my initial proposal for the process is the following:
+
+1. Proposals should be written following the template below and have one Technical Committee (TC) and/or Governance Committee (GC) sponsor, who will regularly provide the TC and GC information about the state of the project.
+1. Once a sponsor is found, the TC and GC will vote on accepting this new project on the Slack channel #opentelemetry-gc-tc. 
+    1. After one week, the voting closes automatically, with the proposal being accepted if it has received at least one :thumbs-up: (that of the sponsor, presumably).
+    1. If at least one :thumbs-down: is given, or a TC/GC member has restrictions about the project but hasn’t given a :thumbs-down:, the voting continues until a majority is reached or the restrictions are cleared.
+    1. The voting closes automatically once a simple majority of the TC/GC electorate has chosen one side.
+1. Proponents should abide by OpenTelemetry’s Code of Conduct (currently the same as CNCF’s).
+1. There’s no expectation that small sandbox projects will have regular calls, but there is an expectation that all decisions will be made in public and transparently.
+1. Sandbox projects do NOT have the right to feature OpenTelemetry’s name on their websites. 
+
+#### Template
+
+> Project name:
+> 
+> Repository name:
+>  
+> Motivation:
```

Comment:
Motivation for the project / product or for joining the sandbox? I would require both.

In general, I think this should be structured to impost as little as necessary overhead on GC/TC and as much as possible on the project authors. If we have specific acceptance criteria / checklist, it should be filled out in the project proposal. See CNCF process - there is an extensive checklist, including for sandbox projects.

Reply:
> Motivation for the project / product or for joining the sandbox? I would require both.

Good idea. I had a "use-case" section before, but removed when I added the example and felt that stating the use-case in the motivation would be enough.